idea/testdata/.config/idea/templates/run-configurations/data-node.env.template

```diff
@@ -0,0 +1,16 @@
+# vim: ft=sh
+GRAYLOG_DATANODE_PASSWORD_SECRET={{ .PasswordSecret }}
+GRAYLOG_DATANODE_NODE_ID_FILE={{ .DataDir }}/node-id
+GRAYLOG_DATANODE_CONFIG_LOCATION={{ .DataDir }}/config
+GRAYLOG_DATANODE_NATIVE_LIB_DIR={{ .DataDir }}/native_libs
+
+GRAYLOG_DATANODE_DATANODE_HTTP_PORT={{ sub 8999 .PortOffset }}
+GRAYLOG_DATANODE_OPENSEARCH_HTTP_PORT={{ add 9200 .PortOffset }}
+GRAYLOG_DATANODE_OPENSEARCH_TRANSPORT_PORT={{ add 9300 .PortOffset }}
+GRAYLOG_DATANODE_OPENSEARCH_HEAP=512m
+GRAYLOG_DATANODE_OPENSEARCH_LOCATION=../graylog-project-repos/graylog2-server/data-node/target/opensearch/opensearch-{{ .OpenSearchVersion }}-linux-x64
```

Comment:
This is a suggestion for the actual template, but then of course also this test template:

```suggestion
GRAYLOG_DATANODE_OPENSEARCH_LOCATION=../graylog-project-repos/graylog2-server/data-node/target/opensearch
```